websites/G/Goodreads/presence.ts

```diff
@@ -0,0 +1,35 @@
+const presence = new Presence({
+	clientId: "765622778317373440"
+}),
+elapsed = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {

```

Comment:
```suggestion
presence.on('UpdateData', async () => {
```

Single quotes to be more consistent.

Reply:
u can use double quotes or even single ones, we are accepting both.